src/audio/buffer.c

```diff
@@ -164,6 +164,49 @@ int buffer_sync_shadow_dp_queue(struct comp_buffer *buffer, size_t limit)
 }
 #endif /* CONFIG_ZEPHYR_DP_SCHEDULER */
 
+struct comp_buffer *buffer_alloc_range(size_t preferred_size, size_t minimum_size, uint32_t caps,
```

Comment:
@softwarecki One stylistic note. Some of your git commits have confused me a bit when you describe the changes in past tense (like "two new fucntions have been added"). My first reaction is that this was done in some earlier commit and this commit somehow builds on top of those older changes. But alas, you are describing what happens in this commit. I think the more common practise is to use present tense in the commit message. One good guide I found: ""If you want to write a git commit message properly, you should use the imperative mood. This means you need to eliminate the temptation to use gerunds or past tense in your subject lines. Don't write a git commit subject line that talks about what you did, or what you are doing. Instead, describe what was done."" (ref: https://www.theserverside.com/video/Follow-these-git-commit-message-guidelines ) . 

Not a showstopper, but just something that might confuse other reviewers as well (I know I stumbled on this in the virtual buffer allocation PR).

Reply:
@kv2019i I corrected the description of the commits trying to use the imperative mood.